src/virtual_lodf_calculations.jl

```diff
@@ -47,61 +54,53 @@ struct VirtualLODF{Ax, L <: NTuple{2, Dict}} <: PowerNetworkMatrix{Float64}
     tol::Base.RefValue{Float64}
 end
 
+"""
+Sets to zero those elements of each LODF matrix row whose absolute values are 
+below the threshold specified by the field "tol".
+
+# Arguments
+- `mat::VirtualLODF`:
+        VirtualLODF structure
+- `tol::Float64`:
+        tolerance
+"""
+function drop_small_entries!(mat::VirtualLODF, tol::Float64)
```

Comment:
I don't know if we should have this function. I think it might be an awkward workflow since it assumes that the user wants to sparsity after having created the rows. It's better we don't support this.

Reply:
ok, I will remove this function